lib/wallet/src/Cardano/Wallet/Write/ProtocolParameters.hs

```diff
@@ -0,0 +1,45 @@
+{-# LANGUAGE NamedFieldPuns #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeApplications #-}
+
+-- |
+-- Copyright: © 2023 IOHK
+-- License: Apache-2.0
+--
+-- Provides protocol parameters.
+--
+module Cardano.Wallet.Write.ProtocolParameters
+    ( ProtocolParameters (..)
+    , unsafeFromWalletProtocolParameters
+    ) where
+
+import Prelude
+
+import qualified Cardano.Api as CardanoApi
+import qualified Cardano.Wallet.Primitive.Types as Wallet
+
+-- TODO: Make this data type abstract: don't export the constructor.
```

Comment:
"TODO: Replace this type with a re-exported `Ledger.PParams era`" 😛

Reply:
@Anviking done in 92977d7858ff46350c421b5d1320776fc94f4275.